tools/fast/fastesting/basic.go

```diff
@@ -61,7 +61,7 @@ func NewTestEnvironment(ctx context.Context, t *testing.T, fastenvOpts fast.File
 
 	fastenvOpts = fast.FilecoinOpts{
 		InitOpts:   append([]fast.ProcessInitOption{fast.POGenesisFile(genesisURI)}, fastenvOpts.InitOpts...),
-		DaemonOpts: append([]fast.ProcessDaemonOption{fast.POBlockTime(50 * time.Millisecond)}, fastenvOpts.DaemonOpts...),
+		DaemonOpts: append([]fast.ProcessDaemonOption{fast.POBlockTime(100 * time.Millisecond)}, fastenvOpts.DaemonOpts...),
```

Comment:
If this actually makes a difference then we are in a very bad world that I thought we had left. Any dependence on wall clock time is **fundamentally flaky**. Tests that pass or fail depending on the load on the machine they are running on are broken. Tests that rely on sequencing of events must use actual synchronisation primitives to ensure that such events have happened. Otherwise we will continually increase this time in a futile attempt to have stable tests, forever paying more and more dev time waiting for them to run and discouraging their use.

The `time.Sleep` in the test that we've disabled is an absolute no-go. That test needs to be rewritten for multiple reasons, of which that is one.

Which tests are sensitive to this block time? If it was only the deal redemption one, please revert this. If you feel it's necessary to commit this change, please file an issue that those tests depend on simulated block time and are a liability.

cc @acruikshank @travisperson

Reply:
@anorth The story behind the change you have notated is: the original block time set across all FAST tests was 1 ms. The original deals show test was consistently failing when creating deals. The message wait was bailing out immediately. It appeared that the context had ended/disappeared before the deal was accepted on chain.  When I passed in a longer block time for the `fast.FilecoinOpts{}`, it failed for an entirely different reason, and an issue was filed for this (#2900).  To get the original test to pass I felt it was harmless to temporarily increase it to 50 ms, which passes locally but is flaky on CI.

These two integration tests for deals show are the only ones I know of sensitive to block time.

Given your comment above, and that the FAST environment setup is designed to allow you to pass in a block time, are you also saying that block time should not be configurable?

Reply:
As discussed, only the `deals show` test has this dependency, so let's make it a parameter so all the other tests can be fast, and the problematic ones easy to identify.

Reply:
Also it will come in a separate PR.